boards/pyportal/src/lib.rs

```diff
@@ -0,0 +1,79 @@
+#![no_std]
+#![recursion_limit = "1024"]
+
+pub mod pins;
+use atsamd_hal as hal;
+
+#[cfg(feature = "rt")]
+pub use cortex_m_rt::entry;
+
+pub use crate::pins::Pins;
+pub use hal::target_device::*;
+pub use hal::*;
+
+use gpio::{Floating, Input, Port};
+use hal::clock::GenericClockController;
+use hal::sercom::{I2CMaster5, PadPin, SPIMaster2};
+use hal::time::Hertz;
+
+#[cfg(feature = "usb")]
+pub use hal::usb::UsbBus;
+#[cfg(feature = "usb")]
+use usb_device::bus::UsbBusWrapper;
+
+/// Convenience for setting up the 2x3 header block for SPI.
```

Comment:
This line of comment is referencing the 2x3 pin header on the Metro, and should be removed.